test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildPackTaskTests.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Xml.Linq;
+using NuGet.Frameworks;
+using NuGet.Packaging;
+using NuGet.ProjectModel;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace Msbuild.Integration.Test
+{
+    [Collection("Msbuild Integration Tests")]
+    public class MsbuildPackTaskTests
+    {
+        private MsbuildIntegrationTestFixture _msbuildFixture;
+
+        public MsbuildPackTaskTests(MsbuildIntegrationTestFixture fixture)
+        {
+            _msbuildFixture = fixture;
+        }
+
+        [PlatformFact(Platform.Windows)]
+        public void MsbuildPack_RequireLicenseAcceptanceNotEmittedWhenUnspecified()
+        {
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+
+                var net461 = NuGetFramework.Parse("net461");
+
+                var projectA = new SimpleTestProjectContext(
```

Comment:
The test project you are using is not SDK based.
Only SDK based projects import the pack SDK by default. Refer to https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package-msbuild. 

There is utility method in the simpletestprojectcontext CreateNETCoreWithSDK. 

Try using that instead